packages/create-sitecore-jss/src/common/Answer.ts

```diff
@@ -5,9 +5,6 @@ export interface Answer extends Answers {
   silent?: boolean;
   appName: string;
   destination: string;
-  fetchWith: string;
-  hostName: string;
```

Comment:
Wouldn't `fetchWith` and `hostName` apply to all templates? Maybe not `fetchWith` if we are now considering react native, but that would mean `appPrefix` should move as well.

Reply:
@ambrauer  We don't need to move `appPrefix` because it's used for each template. `fetchWith` is used only for the base template, same for `hostname`. Did I get correct?

Reply:
Nvm, totally fine for now as this will be refactored in separate work as we discussed.